example_dags/example_google_bigquery_gcs_load_and_save.py

```diff
@@ -0,0 +1,53 @@
+"""
+This Example DAG:
+ - Pulls a CSV file from Github and loads it into BigQuery.
+ - Extracts the data from BigQuery and load into in-memory Pandas Dataframe
+ - Finds the Top 5 movies based on Rating using pandas dataframe
+ - And loads it into a Google Cloud Storage bucket in a CSV file
+
+Pre-requisites:
+ - Create an Airflow Connection to connect to Bigquery Table. Example:
```

Comment:
What are your thoughts on encouraging users to use Airflow default connections as opposed to creating a new one?

Reply:
We recommend in Production use-cases to run `airflow db upgrade` (not `airflow db init`. Ref: [Airflow docs](https://airflow.apache.org/docs/apache-airflow/stable/production-deployment.html#database-backend) ) which does not create default connections.